docs/embedded/getting-started/containertypes.md

```diff
@@ -45,36 +52,113 @@ The following restrictions are applied to trial container types:
 ```
 
+Once the container type is created, add the Azure billing profile.
+
 ```powershell
 Add-SPOContainerTypeBilling –ContainerTypeId <ContainerTypeId> -AzureSubscriptionId <AzureSubscriptionId> -ResourceGroup <ResourceGroup> -Region <Region>
 ```
 
 > [!NOTE]
 > The user or admin who sets up a billing relationship for SharePoint Embedded must have owner or contributor permissions on the Azure subscription.
+>
+> Every container type must have an owning application.
+>
+> A single-owning app can only own one container type at a time.
+>
+> An Azure subscription can be attached to any number of container types.
+
+### Standard Container Type - Pass through billing
+With pass-through billing, consumption-based charges are billed directly to the tenant registered to use the SharePoint Embedded application (consuming tenant). Admins in the developer tenant don't need to set up an Azure billing profile when creating a pass-through SharePoint Embedded container type. 
+
+![Pass Through](../images/2bill521.png)
+
+For container types intended to be directly billed to a customer use the flag `-IsPassThroughBilling`. For the direct to customer billed container type, there's no need to attach a billing profile. 
 
-To create a direct to customer billed, standard container type, use the following cmdlet:
+To create a pass through billing, standard container type, use the following cmdlet:
 
 ```powershell
 New-SPOContainerType [-ContainerTypeName] <String> [-OwningApplicationId] <String> [-ApplicationRedirectUrl] <String> [-IsPassThroughBilling] [<CommonParameters>]
 ```
 
+Once the container type is [registered](../getting-started/register-api-documentation.md) in the consuming tenant, the consuming tenant admin (SharePoint Admin or Global Admin) needs to set up the billing profile in the consuming tenant to use the SharePoint Embedded application.
+
+#### Set Up Billing Profile in Consuming Tenant
+1. In [Microsoft 365 admin center](https://admin.microsoft.com/), select **Setup**, and the view the **Billing and licenses** section. Select **Activate pay-as-you-go services.**
+
+    ![Microsoft 365 admin center Files and Content](../images/SyntexActivatePAYGSetup.png)
+
+2. Select **Go to Pay as you go services**.
+3. Select **Apps** under **Syntex services for**, select **Apps** and **SharePoint Embedded**
+ 
+    ![Microsoft 365 admin center SharePoint Embedded Billing setting](../images/SyntexPAYGActivateSPE.png)
+
+    > [NOTE]
+    The subscription configured in the Syntex services will reflect the consuming charges in the Azure billing portal.
+
+4.  Grant admin access on the consuming tenant.  To interact with containers in a consuming tenant, the owning tenant must first be granted permissions by the consuming tenant admin.  This step is crucial as it controls permissions and prevents access denied errors when invoking other APIs.
```

Comment:
This has nothing to do with setting up billing profile in the consuming tenant. You are documenting the Container Type registration flow. So steps 4 and 5 should be the first steps in the doc linked in step 6 (registering container type). Super confusing to have them here.